airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -26,6 +26,8 @@
 Changelog
 ---------
 
+* ``Fix revert default waiter max attempts to sys.maxsize in 'EcsRunTaskOperator' (partial revert of #30586) (#33712)``
```

Comment:
Lets modify this to a more explanatory text.
Dont mention PR numbers. The commit title that we add during release cycle will do that and refer to this PR for more details.

What we are seeking in the note is just a simple explnatipn of the situation "A bug intoduced in provider version... Caused.... In this version we are fixing it by returning value to ..."
Something like this.